charts/alertmanager/templates/_helpers.tpl

```diff
@@ -53,24 +53,26 @@ Return  the proper Storage Class
 {{/*
 Helm 2.11 supports the assignment of a value to a variable defined in a different scope,
 but Helm 2.9 and 2.10 does not support it, so we need to implement this if-else logic.
+This refactoring prioritizes persistence.storageClassName over global.storageClass.
```

Comment:
This comment will need to be removed, or updated for context outside of this PR.

Reply:
done.